create.sh

```diff
@@ -50,7 +50,7 @@ fi
 # TODO: if the user interrupts the process here, the AAD application will leak.
 
 cat >_data/manifest.yaml <<EOF
-name: openshift
+name: $RESOURCEGROUP
```

Comment:
Operator etcd uses certificates with `master-etcd.namespace.svc` to communicate. Those names need to be in the certificates. We need to know name for those. Could we use manifest name field for a namespace?

Reply:
Hrm, they assume the operator runs on a different namespace than etcd?

Reply:
this should not be even a case. I assume because operator can be global or local it uses same code base. And we end with full DNS names.
```
 etcdserver: not healthy for reconfigure, rejecting member add {ID:e6a985970d419f37 RaftAttributes:{PeerURLs:[https://master-etcd-pkgtnptbmp.master-etcd.mjudeikis-hcp.svc:2380]} Attributes:{Name: ClientURLs:[]}}
2018-07-13 10:47:54.104837 W | etcdserver: timed out waiting for read index response
2018-07-13 10:48:01.390910 I | etcdserver/membership: added member 48e34665e30d3732 [https://master-etcd-m9glrkvqm2.master-etcd.mjudeikis-hcp.svc:2380] to cluster 6b80ae4a9ed28414
```
it advertises as full fqdn:
```
  - command:
    - /usr/local/bin/etcd
    - --data-dir=/var/etcd/data
    - --name=master-etcd-5lswnbnvvb
    - --initial-advertise-peer-urls=https://master-etcd-5lswnbnvvb.master-etcd.mjudeikis-hcp.svc:2380
    - --listen-peer-urls=https://0.0.0.0:2380
    - --listen-client-urls=https://0.0.0.0:2379
    - --advertise-client-urls=https://master-etcd-5lswnbnvvb.master-etcd.mjudeikis-hcp.svc:2379
    - --initial-cluster=master-etcd-5lswnbnvvb=https://master-etcd-5lswnbnvvb.master-etcd.mjudeikis-hcp.svc:2380
```
Where advertise peer url is:
```
func (m *Member) Addr() string {
	return fmt.Sprintf("%s.%s.%s.svc", m.Name, clusterNameFromMemberName(m.Name), m.Namespace)
}
```

Reply:
We may need to loop in someone from the etcd team. I would expect both etcd peers and the operator could assume they can run on the same namespace.